dali/python/nvidia/dali/ops.py

```diff
@@ -394,7 +410,8 @@ def __init__(self, inputs, op, **kwargs):
                 self._inputs = list(self._inputs) + [arg_inp]
 
         if self._op.schema.IsDeprecated():
-            use_instead = self._op.schema.DeprecatedInFavorOf()
+            # TODO(klecki): how to know if this is fn or ops?
+            use_instead = _op_name(self._op.schema.DeprecatedInFavorOf(), "fn")
             msg = "WARNING: `{}` is now deprecated".format(type(self._op).__name__)
             if use_instead:
                 msg +=". Use `" + use_instead + "` instead"
```

Comment:
check if you have a deprecation message and use it here

Reply:
done